### ISSUE TITLE
Plugin - 2D MPR

### ISSUE THREAD

**Issue by cylien** [cylab-tw/bluelight#31]
Creating a display protocol for 2D MPR with 3 columns.

![圖片](https://github.com/cylab-tw/bluelight/assets/3929275/dcfee487-3386-44d8-8193-401de1e2e4ae)

**Comment by cmylmz0** [comment]
It's a very nice and necessary plugin. When will it be completed?

**Comment by birdeggb2777** [comment]
@cmylmz0 I think it will be done in a few days.

**Comment by cylien** [comment]
Added it.
177c612
bce65ce

**Comment by cmylmz0** [comment]
We can't draw rtss in this plugin you have made. We can see the drawings we have done before. I want to draw RTSS on Sagittal and Frontal images as well. 

In other words, I want to make an instant RTSS drawing on the plugin you have made now. How can I do it?

**Comment by cylien** [comment]
You can't draw RTSS on Sagittal and Frontal images in this plugin ([mpr2.js](https://github.com/cylab-tw/bluelight/blob/master/bluelight/scripts/plugin/mpr2.js)). 

We will discuss how to draw RTSS objects on that two views to plan the further fetures about this plugin.

By the way, we also have the annotation tools to draw annotations on images. Please see this [video](https://www.youtube.com/watch?v=N2VLWxpTWjg)

In another Plugin ([mpr.js](https://github.com/cylab-tw/bluelight/blob/master/bluelight/scripts/plugin/mpr.js)):  RTSS can be shown on the images. 
![圖片](https://github.com/cylab-tw/bluelight/assets/3929275/31470b81-4c8a-44d6-922e-fbdabbd65f5a)

**Comment by cmylmz0** [comment]
I know other tools thank you @cylien. As I said, what is required is to be able to draw RTSS on the other 2 views. I would be very happy if you can do this.

**Comment by cmylmz0** [comment]
> I know other tools thank you @cylien. As I said, what is required is to be able to draw RTSS on the other 2 views. I would be very happy if you can do this.

Hi @cylien and @birdeggb2777. Any new update on the plugin?

**Comment by cylien** [comment]
@cmylmz0 we did not discuss this yet. Please wait for us, thanks.

**Comment by cylien** [comment]
@cmylmz0 , After internal discussion, We decided that BlueLight will not support drawing RTSS on Sagittal and Frontal images. Because BlueLight focus on the efficient display of the DICOM object in the browser.  We have evaluated this feature will cost massive memory in the browser which has a very high risk of triggering a browser crash. We think this function may be suitable to implement in a native application such as 3DSlicer, Radiant, etc.
